docs/users_guide/sentiment-analysis.md

```diff
@@ -0,0 +1,76 @@
+[issue-template]: ../../../issues/new?template=BUG_REPORT.md
+[feature-template]: ../../../issues/new?template=FEATURE_REQUEST.md
+
+![singnetlogo](../assets/singnet-logo.jpg?raw=true 'SingularityNET')
+
+# Sentiment Analysis Service
+
+It is part of our [NLP Services](https://github.com/singnet/nlp-services).
+
+### Welcome
+
+The service receives as sentences to be analyzed it's sentiment.
+
+### What’s the point?
+
+The service process input sentences and returns the analyzed sentence. 
+
+### How does it work?
+
+The user must provide the following inputs:
+
+ - `value`: text sentence.
+ 
+
+The output format is:
+ - `value`: analyzed sentences in base64.
+ 
+You can use this service from [SingularityNET DApp](http://beta.singularitynet.io/).
+
+You can also call the service from SingularityNET CLI (`snet`).
+
+Assuming that you have an open channel (`id: 0`) to this service:
+
+##### Input data example:
+
+For this example sentence as input data:
+```
+Great price, fast shipping, great product.
+
+@Olielayus I want to go to promote GEAR AND GROOVE but unfornately no ride there  I may b going to the one in Anaheim in May though.
+
+@maja_dren2, is still sick, and worrying the orange she just ate is going to come back up... ugh. 
+```
+
+Ps: You must break line between the sentences before encoding
+
+Encode in base64 utf-8 text and the result will be like this:
```

Comment:
I still don't understand why this is required? base64 was a hack for sending binary with json-rpc, we should be removing it for grpc.

Reply:
This is legacy code from json-rpc service version. I'll be improving part of code soon.